frontend/mr-admin-flate/src/components/statuselementer/Tiltakstypestatus.tsx

```diff
@@ -1,25 +1,21 @@
-import { Tag } from "@navikt/ds-react";
-import { Tiltakstype } from "mulighetsrommet-api-client";
-import { kalkulerStatusBasertPaaFraOgTilDato } from "../../utils/Utils";
+import {Tag} from "@navikt/ds-react";
+import {Tiltakstype} from "mulighetsrommet-api-client";
```

Comment:
Kan være greit å stille inn intellij til å kjøre `prettier` (siden dette brukes til frontend-koden) som default formatter for js/ts, så slipper man denne frem-og-tilbake formatteringen.

Et oppsett som kan funke:

![image](https://user-images.githubusercontent.com/1423083/220679836-63d1ef41-e272-4d40-b543-6aa88d769e4a.png)